friends/src/App.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+import './App.css';
+import { Route } from 'react-router-dom';
+import axios from 'axios';
+
+import Friends from './Components/friends';
+import styled from 'styled-components';
+import AddFriend from './Components/addFriendForm';
+
+const AppContainer = styled.div `
+max-width: 500px;
+width:100%;
+display: flex;
+margin: 0 auto;
+margin-top: 60px;
+`
+
+
+
+class App extends Component {
+  constructor(props){
+    super(props);
+    this.state={
+       friendsData: [],
+           name: '',
+           age: null,
+           email: '',
+           modal: false,
+    }
+    this.toggle = this.toggle.bind(this);
+}
+
+componentDidMount = () => {
+axios
+.get('http://localhost:5000/friends')
+.then(response => {
+   this.setState({friendsData: response.data})
+   console.log(this.state.friendsData)
+})
+}
+  
+
+  toggle() {
```

Comment:
If you turn this into an ES6 Arrow function, you won't need to bind it in the constructor